tests/Scout/ScoutEngineTest.php

```diff
@@ -31,6 +34,82 @@ class ScoutEngineTest extends TestCase
 {
     private const EXPECTED_TYPEMAP = ['root' => 'object', 'document' => 'bson', 'array' => 'bson'];
 
+    public function testCreateIndexInvalidDefinition(): void
+    {
+        $database = m::mock(Database::class);
+        $engine = new ScoutEngine($database, false, ['collection_invalid' => ['foo' => 'bar']]);
+
+        $this->expectException(LogicException::class);
+        $this->expectExceptionMessage('The search index definition for collection "scout.mongodb.index-definition.collection_invalid" must contain a "mappings" key');
```

Comment:
Does this attempt to match the entire message, or is it only a prefix?

Reply:
`str_contains` is used by PHPUnit for this.